### ISSUE TITLE
Reason for pymummer<=0.10.3 ?

### ISSUE THREAD

**Issue by tseemann** [sanger-pathogens/ariba#267]
```
ERROR: ariba 2.13.5 has requirement pymummer<=0.10.3, but you'll have pymummer 0.11.0 which is incompatible.
```
Is this restriction required?

**Comment by kpepper** [comment]
Yes, I had compatibility issues with the latest 0.11.0 pymummer release (unit test failures from memory). The version restriction is a temporary fix, until there's time to work out what the cause is.

**Comment by satta** [comment]
This is due to a change in how the variant strand is stored, see https://github.com/sanger-pathogens/pymummer/commit/8d44dd1178aa0858155a5778cd374e0d7568d662. 

We have a patch in Debian that addresses the test failures here: https://salsa.debian.org/med-team/ariba/-/blob/master/debian/patches/support-pymummer-0.11.patch
